client_secrets.json

```diff
@@ -0,0 +1,9 @@
+{
+  "web": {
+    "client_id": "602021688774-lrtke5pp9kvekdpovvcamv5q5vo3hhfv.apps.googleusercontent.com",
+    "client_secret": "m6MddZ9oFfjS-cFbOVfnnNrc",
```

Comment:
Nit, you probably wouldn't submit this in the clear to a public repository. I don't think it's an issue in the scope of this project.